cmd/statusd/main.go

```diff
@@ -184,3 +205,25 @@ Options:
 	fmt.Fprintf(os.Stderr, usage) // nolint: gas
 	flag.PrintDefaults()
 }
+
+func haltOnInterruptSignal(nodeManager common.NodeManager) {
```

Comment:
Would like a semantic comment for this (any) new function, regardless if private or public. Simply addressed to the poor maintaining dev in one year. :wink: